index.html

```diff
@@ -182,43 +242,87 @@ <h1 id="t_about">About me</h1>
-                <li>Html</li>
-                <li>Css</li>
-              </ul>
-            </li>
-          </ul>
-        
-          <ul id="style2" class="frameworks">
-            <li class="align">Frameworks
-              <ul id="list" class="none">
-                <li>Bootstrap</li>
-                <li>Ruby on Rails</li>
-                <li>RSPec</li>
-                <li>CapyBara</li>
-                <li>Selenium</li>
-              </ul>
-            </li>
-          </ul>
+        <ul id="style1" class="laguage">
+          <li class="align">Language
+            <ul id="list" class="none">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+          </li>
+        </ul>
 
-          <ul id="style3" class="skills">
-            <li class="align">Skills
-              <ul id="list" class="none">
-                <li clas>Codekit</li>
-                <li>Github</li>
-                <li>Codepen</li>
-                <li>Gitlab</li>
-                <li>Terminal</li>
-              </ul>
-            </li>
-          </ul>
+        <ul id="style2" class="frameworks">
+          <li class="align">Frameworks
+            <ul id="list" class="none">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+          </li>
+        </ul>
+
+        <ul id="style3" class="skills">
+          <li class="align">Skills
+            <ul id="list" class="none">
+              <li clas>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </li>
+        </ul>
       </section>
     </section>
 
+    <footer id="form_section">
+      <header>
+        <h1 id="last_header">I'm always interested in hearing about new projects, so if you'd like to chat please get in
+          touch.</h1>
+      </header>
+
+      <form action="https://formspree.io/f/xpznyvjj" method="post">
+        <div id="uName">
+          <label class="bform" for="name">
+            <input class="istyle" type="text" id="name" name="user_name" maxlength="30" placeholder="    Full name"
+              required>
+          </label>
+        </div>
+        <div class="bform" id="uEmail">
+          <label for="email">
+            <input class="istyle" type="email" id="mail" name="user_email" placeholder="    Email address" required>
+          </label>
+        </div>
+        <div>
+          <label for="message">
+            <textarea id="formMessage" class="istyle" name="text" id="message" maxlength="500"
+              placeholder="Enter text here" required></textarea>
+          </label>
+        </div>
+        <div id="formButton">
+          <button class="button11" id="lastBotton" type="submit">Get in touch</button>
+        </div>
```

Comment:
Kindly add the button interactions such as hover, active (when it's pressed), disabled and enable (default state) styling as shown in the Figma template. They are currently not working. :pray:

Reply:
@jerryowusu For sure, We will. Thank you

Reply:
@jerryowusu I'm not sure what you mean when you say that a need to add the enable and disabled states. When do I should put the enable state and when do I need to change that state to disable?

Reply:
Fixed.